console/console-init/ui/src/utils/utils.ts

```diff
@@ -256,6 +256,10 @@ const getJson = (objects: any[]) => {
   return object;
 };
 
+const createDeepCopy = (object: any) => {
```

Comment:
i would like to suggest the name of function like deepCopy. what is your thought.